playbooks/templates/.env.j2

```diff
@@ -1,118 +1,237 @@
+SIA_WALLET_PASSWORD={{ webportal_server_config.sia_wallet_password if (webportal_server_config.sia_wallet_password is defined and webportal_server_config.sia_wallet_password is not none and webportal_server_config.sia_wallet_password != '') else (webportal_server_config.sia_wallet_seed if webportal_server_config.sia_wallet_seed is defined else '') }}
 
-# certificate string that will be used to generate ssl certificates, read more in docker/caddy/Caddyfile
-SSL_CERTIFICATE_STRING={{ webportal_cluster_config.portal_cluster_domain + ', *.' + webportal_cluster_config.portal_cluster_domain + ', *.hns.' + webportal_cluster_config.portal_cluster_domain if (webportal_cluster_config is defined) else webportal_server_config.domain_name }}, *.{{ webportal_server_config.domain_name }}, *.hns.{{ webportal_server_config.domain_name }}
+# Certificate string that will be used to generate ssl certificates, read more in docker/caddy/Caddyfile
+# xxxqqq: Deprecated
+{% if portal_cluster_id is defined %}
+SSL_CERTIFICATE_STRING={{ cluster_domain }}, *.{{ cluster_domain }}, *.hns.{{ cluster_domain }}, *.{{ server_domain }}, *.hns.{{ server_domain }}
+{% else %}
+SSL_CERTIFICATE_STRING={{ server_domain }}, *.{{ server_domain }}, *.hns.{{ server_domain }}
+{% endif %}
 
-# (optional) the port on which siad is listening, defaults to 9980
+# (optional) The port on which siad is listening, defaults to 9980.
 API_PORT={{ webportal_server_config.sia_api_port }}
 
-# this is auto generated secure key for your handshake service integration
+# Auto generated secure key for your Handshake service integration.
 HSD_API_KEY={{ webportal_server_config.hsd_api_key }}
 
 ###############################################################################
 # Common
 
-# absolute url to the portal api ie. https://siasky.net (general portal
-# address)
-# xxx update for cluster
-SKYNET_PORTAL_API=https://{{ webportal_server_config.domain_name }}
-
-# this is the administrator contact email you need to supply for communication
-# regarding SSL certification
+# Absolute url to the (cluster) portal API.
+# For a single server: https://single-portal.com
+# For a cluster: https://cluster.com
+SKYNET_PORTAL_API=https://{{
+    cluster_domain
+    if portal_cluster_id is defined
+    else server_domain
+}}
+
+# Administrator contact email.
+# Needed for communication regarding SSL certification.
+# xxxqqq: Deprecated.
 EMAIL_ADDRESS={{ skynetlabs_devs_email }}
 
-# (optional) if using cloudflare as dns loadbalancer (need to change it in
-# Caddyfile too)
+# (optional) If using CloudFlare as DNS loadbalancer (need to change it in
+# Caddyfile too).
+# xxx: Not implemented in Ansible
```

Comment:
nit: `TODO` is more common than `xxx` to know that this is something that needs a follow up.